test/jasmine/tests/annotations_test.js

```diff
@@ -138,9 +138,14 @@ describe('annotations autosize', function() {
             var fullLayout = gd._fullLayout;
             var PREC = 1;
 
+            // xaxis2 need a bit more tolerance to pass on CI
+            // this most likely due to the different text bounding box values
+            // on headfull vs headless browsers.
+            var PREC2 = 0.1;
```

Comment:
> Ah I guess precision is supposed to be a number of digits, like the built-in toBeCloseTo.

Exactly. Maybe we should add an array version of [`toBeWithin`](https://github.com/plotly/plotly.js/blob/8745a22e42933b48b85ae05c7daf46bc20e7726c/test/jasmine/assets/custom_matchers.js#L120) down the road too.